sound/hybrid_font.h

```diff
@@ -72,7 +72,7 @@ class HybridFont : public SaberBase {
     font_config.ReadInCurrentDir("config.ini");
     STDOUT.print("Activating ");
     // TODO: Find more reliable way to figure out if it's a monophonic or polyphonic font!!!!
-    monophonic_hum_ = SFX_poweron || SFX_poweroff || SFX_pwroff || SFX_blast;
+    monophonic_hum_ = SFX_poweron || SFX_poweroff || SFX_pwroff || SFX_stun;
```

Comment:
how about something like this:

 monophonic_hum_ = (SFX_in || SFX_out) ? false : (SFX_poweron || SFX_poweroff || SFX_pwroff || SFX_blast);